packages/components/src/components/code-snippet/_code-snippet.scss

```diff
@@ -124,13 +124,14 @@
     display: inline-block;
 
     &:hover {
-      background-color: $field-01;
+      background-color: $field;
       cursor: auto;
     }
   }
 
   .#{$prefix}--snippet--light.#{$prefix}--snippet--inline.#{$prefix}--snippet--no-copy:hover {
     background-color: $field-02;
+    // this???
```

Comment:
I've just been leaving comments anywhere we reference `$field-02` to come back and update once we figure out the inline-theming approach

Reply:
ahh, good point. I'll update the comments to say `//TO-DO:`